docs/semgrep-app/integrations.md

```diff
@@ -8,12 +8,97 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Integrations
 
+Semgrep App contains third party integrations to allow users to add data from Semgrep to other tools that are part of their workflows.
+
+Currently, Semgrep App integrates with the following tools:
+
+| Tool | Tier availability |
+| ---- | ---------------- |
+| Slack | Community (Free) |
+| Email | Community (Free) |
+| Jira | Team |
+| Amazon S3 | Team |
+| Webhook | Team |
+
+## Where are integrations found?
+
+After logging into your Semgrep App account, you can find **Integrations** as a tab within **Settings.**
+
+![Screenshot of Semgrep's "Create New Integration Channel" menu](../img/integration-firstview.png)<br />
+
+View your saved integration channels by clicking on the **gear** icon within the **Rule board**. Click on the toggles to turn notifications on or off for each channel.
+
+![Screenshot of the Semgrep rule board with integrations](../img/integration-ruleboard.png)<br />
+
+## Slack
```

Comment:
```suggestion
## Integrating various third-party tools

This section describes how to integrate particular third-party tools into Semgrep App.

### Slack
```

Reply:
Again to my first comment - I'm not sure - are we integrating Semgrep App with third-party tools or third-party tools are integrated into Semgrep App? Fix my error in the text below the Integrating various third-party tools if I was mistaken, please.